integrationtests/test_all_examples.py

```diff
@@ -197,16 +197,6 @@ def get_python_files(self):
     lambda: not is_wx(),
     "Only support wx",
 )
-SEARCHER.skip_file_if(
-    os.path.join(DEMO, "Misc", "demo_group_size.py"),
-    is_qt6,
-    "enable tries to import a missing constant. See enthought/enable#891",
-)
-SEARCHER.skip_file_if(
-    os.path.join(DEMO, "Standard_Editors", "VideoEditor_demo.py"),
-    lambda: not is_qt5(),
-    "Only supported on Qt5",
-)
```

Comment:
And this goes away because Qt4 is no longer in the matrix?

Reply:
Yep